use/posts.js

```diff
@@ -110,32 +110,33 @@ export const usePosts = () => {
   const postFields = [
     {
       name: 'username',
+      label: t('posts.username.label'),
       admin: true,
     },
     {
       name: 'type',
       component: 'TInputSelect',
+      label: t('posts.type.label'),
+      type: 'select',
       options: postTypeList,
     },
     {
       name: 'title',
       hideLabel: true,
-      placeholder: 'Title',
+      placeholder: t('posts.feildTitle.placeholder'),
```

Comment:
You already had it fixed. Why is it again in plural?

```suggestion
      placeholder: t('post.feildTitle.placeholder'),
```